src/sentry/api/serializers/models/event.py

```diff
@@ -134,6 +139,10 @@ def serialize(self, obj, attrs, user):
             'dateCreated': obj.datetime,
             'dateReceived': received,
             'errors': errors,
+            'fingerprints': map(
+                md5_from_hash,
+                get_hashes_for_event(obj)
+            ),
```

Comment:
This matches the logic here: https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/unmerge.py#L140-L143